base/random.jl

```diff
@@ -133,6 +149,18 @@ function srand(r::MersenneTwister, seed::Vector{UInt32})
 end
 
 # MersenneTwister jump
+
+"""
+    randjump(r::MersenneTwister, jumps::Integer, [jumppoly::AbstractString=dSFMT.JPOLY1e21]) -> Vector{MersenneTwister}
+
+Create an array of the size `jumps` of initialized `MersenneTwister` RNG objects where the
+first RNG object given as a parameter and following `MersenneTwister` RNGs in the array
+initialized such that a state of the RNG object in the array would be moved forward (without
+generating numbers) from a previous RNG object array element on a particular number of steps
+encoded by the jump polynomial `jumppoly`.
```

Comment:
`randjump` - tough to digest as one giant sentence, would be clearer if split up a bit